mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -1867,4 +1997,12 @@ trait Completions { this: MetalsGlobal =>
     }
   }
 
+  def isScaladocCompletion(pos: Position, text: String): Boolean = {
+    val line =
+      try {
+        text.split(System.lineSeparator())(pos.line - 1)
+      } catch { case _: Throwable => "" }
```

Comment:
Do we need to catch `Throwable` here? It's probably fine if we catch `NonFatal` instead 

```suggestion
      } catch { case NonFatal(_) => "" }
```

Also, what about wrapping the entire method in `try/catch` and directly returning `false` instead?